src/pages/en/install/auto.md

```diff
@@ -129,31 +86,12 @@ Astro will listen for live file changes in your `src/` directory, so you will no
 
 If you aren't able to open your project in the browser, go back to the terminal where you ran the `start` command to see what went wrong.
 
-## 4. Deploy to the web
-
-It's time to deploy your project to the web! Run the `build` command in your project to build your static website to a new `dist/` folder in your project.
-
-```bash
-# npm
-npm run build
-
-# yarn
-yarn build
-
-# pnpm
-pnpm run build
-```
-
-When the command finishes, you should have a new `dist/` folder in your project that you can deploy directly to your favorite web host.
-
-To get started hosting your website for free, check out our proud hosting partner, [Netlify](https://www.netlify.com/). For instructions on deploying to whatever host you choose, read our detailed [deployment guide](/en/guides/deploy/).
-
```

Comment:
See comment above, which is kind of the same conversation about "where do we draw the line for the installation tutorial?". I'll resolve this thread so that we keep the convo to one thread, but happy to revisit this based on that outcome.